Gemfile

```diff
@@ -1,3 +1,27 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gemspec
+
+
+
+def location_for(place, fake_version = nil)
+  if place =~ /^(git:[^#]*)#(.*)/
+    [fake_version, { :git => $1, :branch => $2, :require => false }].compact
+  elsif place =~ /^file:\/\/(.*)/
+    ['>= 0', { :path => File.expand_path($1), :require => false }]
+  else
+    [place, { :require => false }]
+  end
+end
+
+
+# We don't put beaker in as a test dependency because we
+# don't want to create a transitive dependency
+group :system_tests do
+  gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 2.0')
```

Comment:
And we're doing this because we're relying on functionality in Beaker being present at test time -- because our library is a module that gets included into Beaker and we rely on things like `block_on` being defined for us at test time.

How do we expect to run our system tests then -- i.e., to make this gem group active in the context when we want to run our tests?